### PR TITLE
feat: add solution for day 16-09-2024

### PR DESCRIPTION
**Pull request by duyen-nt**

### CODE REVIEW

File: daily-challenges/2024-09/16/duyennt.spec.ts [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+  console.log(reverseString("hello")); 
+  
+  //  Playwright
+  // Viết code automation cho test case sau (có thể sử copy code từ bài trước để code nhanh hơn)
+  // - Đi tới trang: https://material.playwrightvn.com/
+  // - Click vào: Bài học 1: Register Page (có đủ các element)
+  // - Điền vào đầy đủ các thông tin của user
+  // - Kiểm tra kết quả đúng như thông tin đã điền.   
+  
+  import { test, expect } from '@playwright/test';
```
Comment (minhphong306) [inline daily-challenges/2024-09/16/duyennt.spec.ts:38]: Đưa phần code playwright ra khỏi function.

**Review by minhphong306 — CHANGES_REQUESTED** [review]
Please fix comments.

File: daily-challenges/2024-09/16/duyennt.spec.ts (hunk 1 above)
Comment (duyen-nt) [reply daily-challenges/2024-09/16/duyennt.spec.ts]: Em đã fix ạ

**Review by minhphong306 — APPROVED** [review]
LGTM

**Pull request merged**